igvc_navigation/launch/octomap.yaml

```diff
@@ -0,0 +1,76 @@
+octree:
+    resolution: 0.2 # width & length of each grid cell in m
+    clamping:
+        max: 0.95 # Value of octree considered 100% occupied stable
+        min: 0.05 # Value of octree considered 0% occupied stable
+# P(x | x) where x is either miss or hit. So prob_miss: 0.2 means that I'm 20% confident a miss is a miss
+probability_model:
+    lidar:
+        scan:
+            prob_miss: 0.55
+            prob_hit: 0.9
+        ground:
+            prob_miss: 0.6
+            prob_hit: 0.6 # Not used
+        free_space:
+            prob_miss: 0.52
+            prob_hit: 0.6 # Not used
+    camera:
+        prob_miss: 0.45
+        prob_hit: 0.9
+sensor_model:
+    max_range: 5
+    angular_resolution: 0.01
+filters:
```

Comment:
This feels like this should be under the lidar namespace. If you are going to use namespaces group things logically.

Reply:
The filters part? I think it seems like a separate part, considering that this should have been a separate nodelet.